.env.example

```
@@ -157,6 +163,7 @@ LITELLM_BASE_URL=http://localhost:11434
 LITELLM_API_KEY=                              # Only needed if using cloud LiteLLM
 
 # External LLM API Keys (fallback providers - Production: use .secrets.env.sops)
+OPENAI_BASE_URL=                              # Optional OpenAI-compatible upstream, e.g. edgeai Bifrost /v1
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)